pkg/cephfs/volumemounter.go

```diff
@@ -70,7 +70,7 @@ func loadAvailableMounters() error {
 		if err != nil {
 			return err
 		}
-		if majorVers >= 4 && minorVers >= 17 {
+		if (majorVers >= 4 && minorVers >= 17) || majorVers >= 5 {
```

Comment:
Can this be changed to major>4 || (major >=4 & & minor >=17)

Reply:
Done, thanks for the feedback!